cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -33,6 +33,8 @@ spec:
         volumeMounts:
           - name: config
             mountPath: /etc/config
+      nodeSelector:
+        beta.kubernetes.io/masq-agent-ds-ready: "true"            
```

Comment:
But I don't see that it was added to https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-test.sh